sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -36,6 +35,7 @@
  */
 public class ClientLogger {
     private final Logger logger;
+    private final boolean isFromEnv;
```

Comment:
Is this being used by anything other than the instantiation in the constructor? If not let's remove it.

Reply:
Done.